### PR TITLE
add doc for v5.1: Deploy -> Monitor and Alert

### PR DESCRIPTION
**Pull request by marsishandsome**
### What is changed?

TiKV's official website is planned to be refactored as discussed in https://github.com/tikv/website/issues/227.

This PR covers `Deploy ` -> `Monitor and Alert`

### Any related PRs or issues?

https://github.com/tikv/website/issues/227.

### Which version does your change affect?
v5.1

### CODE REVIEW

**Comment by marsishandsome** [comment]
@andylokandy @zz-jason @dcalvin @iosmanthus PTAL

File: content/docs/5.1/deploy/monitor/framework.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "5.1":
+        parent: Monitor and Alert
+        weight: 1
+---
+
+The TiKV monitoring framework adopts two open-source projects: [Prometheus](https://github.com/prometheus/prometheus) and [Grafana](https://github.com/grafana/grafana). TiKV uses Prometheus to store the monitoring and performance metrics, and uses Grafana to visualize these metrics.
+
+## About Prometheus in TiKV
+
+As a time series database, Prometheus has a multi-dimensional data model and flexible query language. Prometheus consists of multiple components. Currently, TiKV uses the following components:
```
Comment (andylokandy) [inline content/docs/5.1/deploy/monitor/framework.md:14]:
```suggestion
Prometheus is a time-series database and has a multi-dimensional data model and flexible query language. Prometheus consists of multiple components. Currently, TiKV uses the following components:
```

File: content/docs/5.1/deploy/monitor/alert.md [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+description: Learn the alert rules in a TiKV cluster.
+menu:
+    "5.1":
+        parent: Monitor and Alert
+        weight: 6
+---
+
+This document describes the alert rules for different components in a TiKV cluster, including the rule descriptions and solutions of the alert items in TiKV, PD, Node_exporter and Blackbox_exporter.
+
+According to the severity level, alert rules are divided into three categories (from high to low): emergency-level, critical-level, and warning-level. This division of severity levels applies to all alert items of each component below.
```
Comment (andylokandy) [inline content/docs/5.1/deploy/monitor/alert.md:12]:
```suggestion
Alert rules are divided into three categories according to the severity level (from high to low): emergency-level, critical-level, and warning-level. This division of severity levels applies to all alert items of each component below.
```

**Comment by marsishandsome** [comment]
@andylokandy PTAL

**Review by andylokandy — APPROVED** [review]

**Review by zz-jason — APPROVED** [review]
LGTM

**Pull request merged**